docs/current_docs/quickstart/snippets/simplify/php/src/HelloDagger.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DaggerModule;
+
+use Dagger\Attribute\DaggerFunction;
+use Dagger\Attribute\DaggerObject;
+use Dagger\Attribute\Doc;
+use Dagger\Container;
+use Dagger\Directory;
+
+use function Dagger\dag;
+
+#[DaggerObject]
+class HelloDagger
+{
+    #[DaggerFunction]
+    #[Doc('Publish the application container after building and testing it on-the-fly')]
+    public function publish(Directory $source): string
+    {
+        $this->test($source);
+
+        return $this
+            ->build($source)
+            ->publish('ttl.sh/myapp-'.rand(0, 10000000));
```

Comment:
Consistency nitpick:

Either all should be `ttl.sh/myapp` or all `ttl.sh/hello-dagger`

https://github.com/dagger/dagger/blob/23c2a97598249a0301c7ac32b41d0a89c99863f6/docs/current_docs/quickstart/snippets/simplify/go/main.go#L21

https://github.com/dagger/dagger/blob/23c2a97598249a0301c7ac32b41d0a89c99863f6/docs/current_docs/quickstart/snippets/simplify/python/__init__.py#L14

https://github.com/dagger/dagger/blob/23c2a97598249a0301c7ac32b41d0a89c99863f6/docs/current_docs/quickstart/snippets/simplify/typescript/index.ts#L12